tests/test_interops/test_ctc.py

```diff
@@ -64,44 +63,24 @@ def create_mock_data(
 
 
 @pytest.mark.parametrize(
-    "cli,is_gt,tczyx",
-    list(itertools.product([False, True], [True, False], [True, False])),
+    "is_gt,tczyx",
```

Comment:
i know it's not from this PR... but it would be a bit more idiomatic/clear to use two params, instead of itertools:

```python
@pytest.mark.parametrize("is_gt", [True, False])
@pytest.mark.parametrize("tczyx", [True, False])
```